spec/README.md

```diff
@@ -1,3 +1,102 @@
+# Testing styleguide
+
+We use ["Futurelearn style" acceptance tests](https://about.futurelearn.com/blog/how-we-write-readable-feature-tests-with-rspec). This style was originally from [the "apply" team's spec docs](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/ba9125e9aa5f212c18b63c7c3d49f8785486df83/docs/testing-styleguide.md)
+
+This style guide is newer than the codebase, so you will see a lot of `let` statements around. Use the boy-scout rule and update the style as you touch things.
+
+## Rules
+
+1. Use a single scenario per file. This prevents the files from becoming too large. Separate logical blocks of steps with newlines.
+2. Use instance variable to carry state between steps. Don't use `let` or `before` blocks.
```

Comment:
I'm not entirely clear on why this is one rule, I think it should be two. e.g.:

2. Create your data in the `given_` block instead of using `let` statements.
3. Use instance variables to carry data and state between steps.